### PR TITLE
[entity6010] Develop Static Price List Interface

### PR DESCRIPTION
**Pull request by jeznorth**
*Issue #:*
https://github.com/bcgov/entity/issues/entity6010

Implement static content / route for initial price list (aka 'Fee Schedule')

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: auth-web/src/views/auth/PriceListView.vue [hunk 1]
```diff
@@ -0,0 +1,181 @@
+<template>
+  <main class="view-container container">
+      <h1 class="mb-4">Price List</h1>
+      <p class="mb-8">Where applicable, tax is included in the price shown. Prices are subject to change without notice.</p>
+      <section class="mb-10">
+        <h2 class="mb-5">Business Registry</h2>
+        <v-card flat>
+          <table>
```
Comment (saravanpa-aot) [inline auth-web/src/views/auth/PriceListView.vue:8]: curious , why not v-data-table?

File: auth-web/src/views/auth/PriceListView.vue [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator'
+import { VueConstructor } from 'vue'
+
+@Component({
+  name: 'PriceListView'
+})
+
+export default class PriceListView extends Vue {
+  data () {
```
Comment (shabeeb-aot) [inline auth-web/src/views/auth/PriceListView.vue:42]: No need of data return. we can use private pricelist:

File: auth-web/src/views/auth/PriceListView.vue (hunk 1 above)
Comment (jeznorth) [reply auth-web/src/views/auth/PriceListView.vue]: No need. Table will suffice here.

File: auth-web/src/views/auth/PriceListView.vue (hunk 2 above)
Comment (jeznorth) [reply auth-web/src/views/auth/PriceListView.vue]: Updated as requested.

File: auth-web/src/views/auth/PriceListView.vue [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+            </tbody>
+          </table>
+        </v-card>
+      </section>
+  </main>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator'
+import { VueConstructor } from 'vue'
```
Comment (shabeeb-aot) [inline auth-web/src/views/auth/PriceListView.vue:77]: Looks like this is not needed. we are not using it anywhere, can you crosscheck?

**Review by shabeeb-aot — APPROVED** [review]

**Pull request merged**